src/js/store/orgs/actions.ts

```diff
@@ -348,3 +353,37 @@ export const commitFailed = ({
     payload: model,
   });
 };
+
+export const refreshOrg = (org: Org): ThunkResult => async (dispatch) => {
```

Comment:
My proposal would be roughly this:

- When we request a refresh (at the beginning of this action), we manually (on the front end, in the reducer) set the `org.currently_refreshing_org` flag to `true`.
- On success, we don't need to do anything -- we've already set the flag we care about.
- On error, we need to unset that flag (and probably display an error toast).

This should be _very_ similar to the action I just wrote in another PR: https://github.com/oddbird/MetaShare/pull/162/files#diff-d69236bdfc813971102a3ad7cdf2bbb9R90-R107